### PR TITLE
Dev iesave report

### PR DESCRIPTION
**Pull request by luizaandrade**
This adds the option to create the report as discussed in dropbox paper

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -43,6 +43,7 @@ _config.yml
 !/**/*.do
 !/**/*.ado
 !/**/*.sthlp
+!*.csv
```
Comment (kbjarkefur) [inline .gitignore:46]:
```suggestion
!/**/output/iesave/*.csv
```

Comment (kbjarkefur) [reply .gitignore:46]: I have many .csv files in the output folder from test runs of other commands that I do not want to commit.

File: src/ado_files/iesave.ado [hunk 2]
```diff
@@ -283,10 +327,426 @@
...
+
+	syntax anything, logfile(string) logname(string) format(string) [debug]
+		
+	if !missing("`debug'") noi di "Entering write_line subcommand"
+	
+	if 		("`format'" == ".csv") local sep ","
+	else if ("`format'" == ".md")  local sep " | "
+								   local line = subinstr(`anything', "~", "`sep'", .) 
+	if		 ("`format'" == ".md") local line   | `line' |
+
```
Comment (kbjarkefur) [inline src/ado_files/iesave.ado:432]:
```suggestion

	*Remove excessive spaces in the line before writing it to file
	local line = trim(itrim("`line'"))
	
```

Comment (kbjarkefur) [reply src/ado_files/iesave.ado:432]: This makes the output files much neater imho and neat is good as the purpose is version control them on a detailed level

**Comment by luizaandrade** [comment]
@kbjarkefur , I have addressed both your comments. Can you please take a look again?

**Comment by kbjarkefur** [comment]
@luizaandrade , looks good. See the chat I sent about the `double`/`float` inconsistency. 

I added code in ca10ded934ed52365e8b55d51e6794991c2cd0ef so that the version number is automatically updated based on what version of the `ietoolkit` package you have installed. Otherwise this will be a pain to update manually.

**Pull request merged**